src/test/java/api/loans/InHouseUseCheckInTest.java

```diff
@@ -120,4 +128,33 @@ public void isNotInHouseUseWhenCheckInServicePointIsNotServingHomeLocation() {
     assertThat(checkInResponse.getJson().containsKey("item"), is(true));
     assertThat(checkInResponse.getInHouseUse(), is(false));
   }
+
+  private void verifyLastCheckInRecordedAsInHouse(UUID itemId, UUID servicePoint) {
```

Comment:
Given that it was decided that we would *not* record that a check in is considered to be in house use, I think this name is misleading.
```suggestion
  private void verifyLastCheckInWasRecorded(UUID itemId, UUID servicePoint) {
```